### PR TITLE
Add info on MyCat provider

### PR DESCRIPTION
**Pull request by rowanmiller**

### CODE REVIEW

File: docs/providers/mycat/index.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+-----------
+
+The following resources will help you get started with this provider.
+ * `Getting started documentation <https://github.com/aspnet/EntityFramework.Docs/issues/252>`_
+ * `Getting started video <https://www.youtube.com/watch?v=q0CXfFNtMZo>`_
+
+Supported Database Engines
+--------------------------
+
+  * MyCat
```
Comment (rowanmiller) [inline docs/providers/mycat/index.rst]: Any particular version restrictions or anything I should list here? For example, on the SQL Server page we list that we work with SQL Server 2008 onwards.

File: docs/providers/mycat/index.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Supported Platforms
+-------------------
+
+  * Full .NET (4.5.1 onwards)
+  * .NET Core
+  * Mono (4.2.0 onwards)
```
Comment (rowanmiller) [inline docs/providers/mycat/index.rst]: @kagamine does it work on Mono?

File: docs/providers/mycat/index.rst [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+Install
+-------
+
+Download and run `the latest release from the project site <https://github.com/PomeloFoundation/Entity-Framework-Core-MyCat-Proxy/releases>`_.
+
+Get Started
+-----------
+
+The following resources will help you get started with this provider.
+ * `Getting started documentation <https://github.com/aspnet/EntityFramework.Docs/issues/252>`_
```
Comment (rowanmiller) [inline docs/providers/mycat/index.rst]: Link to be replaced with content hosted on MyCat site

**Comment by rowanmiller** [comment]
@kagamine could you take a look at this one to ensure it is accurate?

File: docs/providers/mycat/index.rst (hunk 1 above)
Comment (Kagamine) [inline docs/providers/mycat/index.rst]: MyCat supports MySQL, SQL Server 2005+, Oracle, Postgres, MongoDB, etc. 
But when you are using the database which is not MySQL, you cannot use migrations now.
Database creation depends on the specific ef core provider.
Hi, @magicdoom Can you list full the database engines which mycat supported?

File: docs/providers/mycat/index.rst (hunk 2 above)
Comment (Kagamine) [inline docs/providers/mycat/index.rst]: Yes, it works on .net451, mono and .net core

**Comment by thatcat** [comment]
hi @rowanmiller , the information of the database engines which Mycat supported as following: 
Mycat not only supports **Mysql 5.5+**, but also supports **SQLServer 2005+**, **Oracle 9i+**, **PostgreSQL 9.0+**, **MongoDB**, **Sequoiadb**, **Hive** etc.
In the frontend , Application connects to Mycat just like connects to Mysql or SQL Server, and you can use SQL Server native grammar to communicate with Mycat.
In the backend , Mycat supports three kinds of communication methods, they are Mysql native protocol, Postgresql  native protocol and JDBC .

I will link a document about _Mycat for SQL Server_ tomorrow.

**Comment by thatcat** [comment]
@rowanmiller This is the simple introduction document about [Mycat for SQL Server ](https://github.com/MyCATApache/Mycat-doc/blob/master/en/Mycat%20for%20SQL%20Server.docx)

**Comment by EasonFeng5870** [comment]
Hi @rowanmiller , you can get detail informtion from the following file:
https://github.com/MyCATApache/Mycat-doc/blob/master/en/configurations/schema.xml

**Comment by rowanmiller** [comment]
Closing this as we are porting the docs to a new system. Will push changes in the new format once the port is complete.

**Pull request closed without merging**